frontend/src/components/TableSurfingInput.vue

```diff
@@ -23,7 +23,7 @@ export default {
   },
   methods: {
     updateTotal() {
-      this.totalSeconds = this.minutes * 60 + this.seconds + this.penaltyPoints * 10;
+      this.totalSeconds = this.minutes * 60 + this.seconds + this.penaltyPoints;
```

Comment:
Given this change, maybe we should just call it _Penalty seconds_? That would make the use much more obvious to the end-user.